packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/CustomVisConfigTabs.tsx

```diff
@@ -31,18 +110,31 @@ const initVegaLazySchema = async (fields: string[]) => {
         {
             uri: 'https://lightdash.com/schemas/vega-lite-schema-custom.json',
             fileMatch: ['*'],
-            schema: merge(vegaLiteSchema.default, {
-                definitions: {
-                    FieldName: {
-                        type: 'string',
-                        enum: fields,
-                    },
-                },
-            }),
+            schema: merge(vegaLiteSchema.default, {}),
```

Comment:
if definitions are no longer necessary then merge is no longer needed too. I had it inject field autocompletion when referencing